peekingduck/pipeline/nodes/model/hrnetv1/hrnet_model.py

```diff
@@ -37,13 +37,19 @@ def __init__(self, config: Dict[str, Any]) -> None:
         if not 0 <= config["score_threshold"] <= 1:
             raise ValueError("score_threshold must be in [0, 1]")
 
+        weights_dir, model_dir = finder.find_paths(
+            config["root"], config["weights_parent_dir"], config["weights"]
+        )
+
         # check for hrnet weights, if none then download into weights folder
-        if not checker.has_weights(config["root"], config["weights_dir"]):
-            print("---no hrnet weights detected. proceeding to download...---")
-            downloader.download_weights(config["root"], config["blob_file"])
-            print("---hrnet weights download complete.---")
+        if not checker.has_weights(weights_dir, model_dir):
+            self.logger.info(
```

Comment:
Double check if this will cause mlmodel tests to fail since they may not be capturing the logging messages https://github.com/aimakerspace/PeekingDuck/blob/dev/tests/pipeline/nodes/model/hrnetv1/test_hrnet_model.py#L56

\* Similar for other changes.